src/client/dfs/dfs.c

```diff
@@ -625,14 +637,14 @@ check_access(dfs_t *dfs, uid_t uid, gid_t gid, mode_t mode, int mask)
 
 static int
 open_file(dfs_t *dfs, daos_handle_t th, dfs_obj_t *parent, int flags,
-	  daos_oclass_id_t cid, daos_size_t chunk_size, dfs_obj_t *file)
+	  daos_oclass_id_t cid, daos_size_t chunk_size, dfs_obj_t *file,
```

Comment:
minor: but the in/out param is usually the last, so maybe just swap file and len.
same for open_dir, open_symlink

Reply:
Will do.